nab/detectors/htmcore/htmcore_detector.py

```diff
@@ -135,8 +145,10 @@ def handleRecord(self, inputData):
 
   def initialize(self):
     # toggle parameters here
-    #parameters = default_parameters
-    parameters = parameters_numenta_comparable
+    if self.use_optimization:
+      parameters = get_params('params.json')
+    else:
+      parameters = parameters_numenta_comparable
```

Comment:
off topic, but seeing this: maybe I'd like 2 JSON params objects. `params_numenta_comparable` (as close as possbile to NumentaTM detector). And `params_htmcore_best` (our diverged/optimized set).  
It would make later comparisons easier to do.

Reply:
Or maybe using json files for all? So one best_params.json and another numenta_comparable_params.json? Downside would be that we could not add comments I guess.

Reply:
> Downside would be that we could not add comments I guess.

I think the comments are quite useful, atleast for now. 
So maybe the other way, have params_numenta in a file. And params_best in here (and default to them if no cmd arg is passed for the params file)